setup.py

```diff
@@ -29,6 +29,8 @@
     'dpdispatcher>=0.3.11',
     'netCDF4',
     'dargs>=0.2.9',
+    'pybind11>=2.4',
```

Comment:
Why do we need it?

Reply:
Installing `pymatgen-analysis-defects` without `pybind11` will cause errors.